kivy/core/audio/audio_sdl2.pyx

```diff
@@ -42,7 +43,7 @@ cdef mix_init():
         mix_is_init = -1
         return 0
 
-    mix_flags = Mix_Init(0)
+    mix_flags = Mix_Init(MIX_INIT_FLAC|MIX_INIT_MOD|MIX_INIT_MP3|MIX_INIT_OGG)
```

Comment:
AFAIK it just ignores it. My test VM doesn't have MP3 support due to a bug in the sdl packaged with ubuntu 14.04, and the result is that mix_flags does not have MIX_INIT_MP3 set